dialogue-client-test-lib/src/main/java/com/palantir/dialogue/AbstractChannelTest.java

```diff
@@ -308,30 +306,6 @@ public void connectionErrorsSurfaceAsExceptions() throws IOException {
         assertThatThrownBy(call::get).hasCauseInstanceOf(ConnectException.class);
     }
 
-    @Test
-    public void supportsGzipEncryptedResponse() throws Exception {
-        // drain enqueued response
-        channel.execute(endpoint, request).get();
-        server.takeRequest();
-
-        server.enqueue(new MockResponse().addHeader("cOntent-encoding", "gzip").setBody(zip("foo")));
-        Response response = channel.execute(endpoint, request).get();
-        assertThat(response.body()).hasContent("foo");
-        assertThat(server.takeRequest().getHeaders().get("accept-encoding")).isEqualTo("gzip");
-    }
-
-    private static Buffer zip(String content) throws IOException {
-        Buffer gzipBytes = new Buffer();
-        Buffer rawBytes = new Buffer();
-        rawBytes.writeString(content, StandardCharsets.UTF_8);
-        rawBytes.flush();
-
-        GzipSink gzip = new GzipSink(gzipBytes);
-        gzip.write(rawBytes, 3);
-        gzip.close();
-        return gzipBytes;
-    }
```

Comment:
I think we should keep this test, but test all clients wrapped with Channels.create to inherit generalized behavior.